Makefile

```diff
@@ -1,30 +1,32 @@
+REBAR = rebar3
```

Comment:
I would remove the binary altogether. I think that we'll have fewer issues with rebar3 than with the original one.

Do you want me to push the removal of the binary to the PR?